src/test/java/org/hydev/mcpm/client/database/PluginMockFactory.java

```diff
@@ -33,22 +43,51 @@ public static PluginYml meta(String name, String version, String description) {
         );
     }
 
-    public static PluginVersion version(long id, String name, String string) {
-        return new PluginVersion(id, 0, "", meta(name, string, null));
+    /**
+     * Create mock PluginVersion
+     *
+     * @param id ID
+     * @param name Name
+     * @param version Version
+     * @return PluginVersion
+     */
```

Comment:
This is suspicious. Did this not make it into my PR or does this PR need to be rebased?

Either way IIRC empty param descriptions are either IntelliJ or Checkstyle warnings too.

Reply:
yea I didn't know you made it in your pr as well, so I just removed them when rebasing